lib/runtime-core/src/typed_func.rs

```diff
@@ -438,33 +472,91 @@ macro_rules! impl_traits {
             }
         }
 
-        impl< $( $x, )* Rets, Trap, FN > ExternalFunction<( $( $x ),* ), Rets> for FN
+        impl< $( $x, )* Rets, Trap, FN > ExternalFunction<ExplicitVmCtx, ( $( $x ),* ), Rets> for FN
```

Comment:
Implementation of `ExternalFunction<ExplicitVmCtx, …>`, where `FN` must be `Fn(&mut vm::Ctx, …)`. The `wrap` function has a `&mut vm::Ctx` argument.

Note for the ones who aren't familiar with this detail: The `wrap` function will be dereferenced to `*mut vm::Func` (opaque function pointer), and will be used as `FuncPointer` by the `Export` API. So `wrap` acts as a “wrapper” around the user-given function.